### PR TITLE
SEO-172173 Windows Forms Changes

### PR DESCRIPTION
**Pull request by Scholar-jaruha**
Hi @MallikaRK ,
please review the changes made.
Regards,
Scholar.

### CODE REVIEW

File: WindowsForms/ListView/GettingStarted.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ Create a new Windows Forms project in Visual Studio to display the SfListView wi
 ### Adding Control via Designer
 The SfListView control can be added to the application by dragging it from the toolbox and dropping it in designer. The required assembly references will be added automatically.
 
-![Drag and drop the SfListView control into WF application](GettingStarted_images/GettingStarted_img1.png)
+![Drag and drop the SfListView control into WindowsForms application](getting-started_images/windowsforms-drag-drop.png)
```
Comment (MallikaRK) [inline WindowsForms/ListView/GettingStarted.md:25]: WF also fine. Don't change.
Control name missing in image name.
Check and add the control name in all places.

**Review by MallikaRK — CHANGES_REQUESTED** [review]
Commented

**Review by MallikaRK — APPROVED** [review]
Approved

File: WindowsForms/MaskedTextBox/Overview.md [hunk 2]
```diff
@@ -11,7 +11,7 @@ documentation: ug
 The MaskedEditBox control is an edit control that provides an easy and reliable way of collecting user input and displaying standard data in specific formats from any data source. Some of the common uses of the MaskedEditBox controls are IP Addressed it control, Phone Number edit control and Social Security edit control.
 
 
-![Overview of Syncfusion MaskedEditBox](MaskedEditBox-images/MarkedEditBox-img1.png)
+![Overview of Syncfusion MaskedEditBox](MaskedEditBox-images/windowsforms-masked-edit-box-overview.png)
```
Comment (Girija-Rajendran) [inline WindowsForms/MaskedTextBox/Overview.md:14]: image

Comment (Scholar-jaruha) [reply WindowsForms/MaskedTextBox/Overview.md]: changed

**Review by Aishwarya-Ganesan — APPROVED** [review]
Changes are fine.

**Pull request merged**